### DISCUSSION TITLE
Dropdown clear icon to get focus state

### DISCUSSION THREAD

**Discussion by mristov-gb** [primefaces/community#1173] [PrimeNG]
Currently if we try to focus on clear button in dropdown input via **tab** key, there is going into next available html element.
It will be good on **tab** key for accessibility clear icon to be able to get focus state, and with that we can clear our selection.

![image](https://github.com/primefaces/community/assets/150306240/175adb51-0867-4856-af9c-d81c7a13221c)

**Comment by cetincakiroglu** [comment]
Hi,

Close icons are not focusable yet, there isn't a feature to focus on close icons.  However, it might be a good feature to improve DX. We'll wait for community upvotes to implement this feature. 

Thanks for the feedback.
